orbit/models/lgt.py

```diff
@@ -792,7 +792,7 @@ class LGTFull(BaseLGT):
     """
     _supported_estimator_types = [StanEstimatorMCMC, StanEstimatorVI, PyroEstimatorVI]
 
-    def __init__(self, n_bootstrap_draws=None, prediction_percentiles=None, **kwargs):
+    def __init__(self, n_bootstrap_draws=None, prediction_percentiles=[5, 95], **kwargs):
```

Comment:
I'd very strong urge against this. Default args being mutable is generally a bad idea. (I think this is a good description of why: https://docs.python-guide.org/writing/gotchas/)

I'd suggest setting following the design of the otther args and set the default in `_set_default_base_args()`